wren-ui/src/apollo/server/types/diagram.ts

```diff
@@ -70,4 +70,5 @@ export interface DiagramModelRelationField {
   fromColumnName: string;
   toModelName: string;
   toColumnName: string;
+  description: string;
```

Comment:
Should the description be returned In the `transformModelRelationField` method?

Reply:
Already returned here I think

https://github.com/Canner/WrenAI/blob/41d38154ee4a0c73f7ade77703e7882dcf2f96cb/wren-ui/src/apollo/server/resolvers/diagramResolver.ts#L149